clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -168,105 +240,130 @@ bool CheckedRegionFinder::isInStatementPosition(CallExpr *C) {
 }
 
 bool CheckedRegionFinder::VisitCallExpr(CallExpr *C) {
+  // TODO this should really be moved to the other visitor
+  // both for clarity and because it will allow it take advantage of
+  // redundancy checks
```

Comment:
What is "this" that the comment refers to? I see that adding the `_Unchecked` wrapper has indeed moved.

Reply:
Outdated comment.

Reply:
I'm still seeing the comment?! I thought I saw a push, but maybe not...